[2021-05-16 05:53:44,116] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: json_data_ingesation.json_src_cleanup 2021-05-16T05:51:20.017795+00:00 [queued]>
[2021-05-16 05:53:44,126] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: json_data_ingesation.json_src_cleanup 2021-05-16T05:51:20.017795+00:00 [queued]>
[2021-05-16 05:53:44,126] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2021-05-16 05:53:44,126] {taskinstance.py:867} INFO - Starting attempt 1 of 2
[2021-05-16 05:53:44,126] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2021-05-16 05:53:44,136] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): json_src_cleanup> on 2021-05-16T05:51:20.017795+00:00
[2021-05-16 05:53:44,138] {standard_task_runner.py:53} INFO - Started process 3291 to run task
[2021-05-16 05:53:44,185] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: json_data_ingesation.json_src_cleanup 2021-05-16T05:51:20.017795+00:00 [running]> ip-172-31-21-147.ec2.internal
[2021-05-16 05:53:44,196] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-05-16 05:53:44,199] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=json_data_ingesation, task_id=json_src_cleanup, execution_date=20210516T055120, start_date=20210516T055344, end_date=20210516T055344
[2021-05-16 05:53:54,107] {logging_mixin.py:112} INFO - [2021-05-16 05:53:54,107] {local_task_job.py:103} INFO - Task exited with return code 0
